chat_v2_sample/src/main/java/com/zendesk/chat_v2/sample/Global.java

```
@@ -0,0 +1,34 @@
+package com.zendesk.chat_v2.sample;
+
+import android.app.Application;
+
+import com.zendesk.logger.Logger;
+import com.zendesk.util.StringUtils;
+
+import zendesk.chat.Chat;
+
+public class Global extends Application {
+
+    private static final String CHAT_ACCOUNT_KEY = "";
+
+    private static boolean missingCredentials = false;
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        // Enable logging
+        Logger.setLoggable(true);
+
+        if (StringUtils.isEmpty(CHAT_ACCOUNT_KEY)) {
+            missingCredentials = true;
```

Maybe better to throw a runtime exception here to fail early